PROTOCOL.md

```diff
@@ -1942,9 +1942,11 @@ delta-table-version: 123
 ### Read Change Data Feed from a Table
 This is the API for clients to read change data feed from a table.
 
-You can provide either version or timestamp for the start and end of change data feed. The start and end versions and timestamps are inclusive in the queries. To read the changes from a particular start version to the latest version of the table, specify only the starting version or timestamp. Either startingVersion or startingTimestamp must be provided. For either starting or ending parameter, only one parameter (version or timestamp) is supported.
+The API supports a start parameter and and an end parameter. The start/end parameter can either be a version or a timestamp. The start parameter must be provided. If the end parameter is not provided, the API will use the latest table version for it. The parameter range is inclusive in the query.
 
-You specify a version as an long and a timestamps as a string in the format "yyyy-mm-dd hh:mm:ss[.fffffffff]".
+You specify a version as a long and a timestamps as a string in the format "yyyy-mm-dd hh:mm:ss[.fffffffff]".
+
+More related info can be found in [Delta Change Data Feed](https://docs.databricks.com/delta/delta-change-data-feed.html).
```

Comment:
I don't think we should have any link pointing to Databricks specific documentation. At least at this point that documentation still says CDF is only available on Databricks Runtime. It could make ppl think this feature is also somewhat proprietary but it's actually not.

Reply:
This is an interesting point.
If Delta has not open sourced it yet, is it ok for us to open-source this feed via sharing?
Do we need to seek their permission?

Reply:
We've already checked with @zsxwing, CDF will be open source in Q2/Q3.

Good point. I removed the link and added some explanation on the file content in the `CDF File` section.

Reply:
sg!